docs/docs/user/references/editor-integration.md

```diff
@@ -0,0 +1,24 @@
+# Editor integration
+
+KPOps provides JSON schemas that enable autocompletion and validation for some
+of the files that the user must work with.
```

Comment:
is the line break on purpose? I would remove it and let mkdocs take care of that automatically

Reply:
It doesn't break when rendered.

Reply:
hm, I am not a fan of enforcing line breaks in md since this can be easily done on the editor side using linewrap configuration, but it's just a minor cosmetic thing

Reply:
fair enough, removed the line breaks, they only make editing harder anyway